packages/terra-application-navigation/src/common/_PopupCount.jsx

```diff
@@ -8,13 +8,18 @@ import styles from './PopupCount.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * Whether or not the count should be present for height calculations, but hidden from view.
+   * To ensure that row heights are consistent, we need the ability to have the count and not see it.
+   */
+  isPlaceholder: PropTypes.bool,
```

Comment:
Would it make sense to rename to `isHidden`

Reply:
Updated.